drivers/gles2/shaders/scene.glsl

```diff
@@ -10,6 +10,12 @@ precision highp float;
 precision highp int;
 #endif
 
+#ifndef USE_GLES_OVER_GL
+#extension GL_OES_texture_3D : enable
```

Comment:
This will cause a warning message to be printed on devices that don't support ``GL_OES_texture_3D``. However, we are at the max number of conditionals already, so we can't add another one to check for support before enabling the extension.